src/utils/api.js

```diff
@@ -0,0 +1,55 @@
+import axios from "axios";
+
+
+const newsApi = axios.create({
+    baseURL: "https://nc-news-api-4.onrender.com/api"
+})
+
+
+export const getComments= (article_id) => {
+
+    return newsApi.get(`/articles/${article_id}/comments`).then((response)=>{
+        return response.data.comments
+    })
+        
+}
+
+export const getArticleById = (article_id)=>{
+
+    return newsApi.get(`/articles/${article_id}`).then((response)=>{
+        
+        return response.data.article
+    })
+}
+
+
+
+
+
+
+export const upVoteCount = (article_id) => {
+const upVote = { inc_votes: 1 };
```

Comment:
If you move the inc_votes object into each handler in the Votes component, you'd be able to just have the one API call rather than two 🙂